app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StoreCommentRequest;
+use App\Http\Requests\UpdateCommentRequest;
+use App\Models\Comment;
+use App\Models\Post;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
+use Illuminate\Http\Exceptions\HttpResponseException;
+
+
+
+
+
+class CommentController extends Controller
+{
+
+    public function findOrFail($id): Post|JsonResponse
+    {
+        $post = Post::find($id);
+        
+        if (!$post) {
+            throw new HttpResponseException(
+            response()->json([
+                'status' => false,
+                'message' => 'Post Not Found',
+            ], 404)
+        );
+        }
+        
+        return $post;
+    }
```

Comment:
```suggestion
    public function findOrFail($id): Post|JsonResponse
    {
        return Post::findOrFail($id);
    }
```

At this point there's no use for a function 😄